os/include/tinyara/signature.h

```diff
@@ -29,6 +29,9 @@
 #define USER_SIGN_PREPEND_SIZE     0
 #endif
 
+#define SIGNATURE_VERIFY_TRUE      0
+#define SIGNATURE_VERIFY_FALSE     1
```

Comment:
1. Need to apply this to amebad as well.
2. `VERIFY_TRUE` and `FALSE`? `SIGNATURE_VERIFIED` and `SIGNATURE_INVALID`?

Reply:
Amebad returns 0(OK) when verify success, but amebasmart returns 1 when success. So I added SIGNATURE_VERIFY_TRUE / SIGNATURE_VERIFY_FALSE value only for amebasmart. 
This is already applied in our product code for now.

Reply:
2. I'll change the name like this
`SIGNATURE_VALID / SIGNATURE_INVALID`

Reply:
> Amebad returns 0(OK) when verify success, but amebasmart returns 1 when success. So I added SIGNATURE_VERIFY_TRUE / SIGNATURE_VERIFY_FALSE value only for amebasmart.

It should not be. Every chip should provide the same returns. If not, will you modify binary manager for each board?